image_provisioner/inventory/hosts.AWS

```diff
@@ -46,3 +46,5 @@ image_version=v3.3.0.30
 # image naming
 # set openshift_version to the rpm version, sometimes with a trailing -N
 openshift_version=3.3.0.30-1
+# set atomic to True for atomic image
+atomic=False
```

Comment:
Yes, it does now that you've added defaults like I suggested. We can keep the inventory to keep track of it or otherwise just put it only in the docs.